### ISSUE TITLE
German default locale thousands delimiter should be `.`

### ISSUE THREAD

**Issue by rexblack** [adamwdraper/Numeral-js#533]
In german, thousands delimiter is a dot.

https://github.com/adamwdraper/Numeral-js/blob/master/locales/de.js#L16

**Comment by anselms** [comment]
Agreed, but weirdly enough it still parses German numbers correctly, at least for me.

**Comment by leggewie** [comment]
This is a dupe of #139
